honeybee_energy/config.py

```diff
@@ -0,0 +1,322 @@
+"""Honeybee_energy configurations.
+
+Import this into every module where access configurations are needed.
+
+Usage:
+    from honeybee_energy.config import folders
+    print(folders.energyplus_path)
+    print(folders.openstudio_path)
+    folders.energyplus_path = "C:/EnergyPlusV9-0-1"
+"""
+import os
+import sys
+import json
+
+
+class Folders(object):
+    """Honeybee_energy folders.
+
+    Args:
+        mute: If False, the paths to the various folders will be printed as they
+            are found. If True, no printing will occur upon initialization of this
+            class. Default: True.
+
+    Properties:
+        * openstudio_path
+        * energyplus_path
+        * template_library_folder
+        * default_simulation_folder
+        * energy_model_measure_path
+        * python_package_path
+        * python_exe_path
+
+    Usage:
+        folders = Folders(mute=False)
+        print(folders.energyplus_path)
+    """
+
+    def __init__(self, mute=True):
+        # set the mute value
+        self.mute = bool(mute)
+
+        # load paths from the config JSON file within this package
+        config_file = os.path.join(os.path.dirname(__file__), 'config.json')
+        self.load_from_file(config_file)
+
+    @property
+    def openstudio_path(self):
+        """Get or set the path to OpenStudio installation folder.
+        
+        This is the "master" directory for OpenStudio installation (the one that
+        includes EnergyPlus, Ruby, etc.). It must contain a "bin" folder
+        with the openstudio executable in order to be valid.
+        """
+        return self._openstudio_path
+
+    @openstudio_path.setter
+    def openstudio_path(self, path):
+        if not path:  # check the default installation folders
+            path = self._find_openstudio_folder()
```

Comment:
I think we should remove all the find methods and make it explicit as part of the process by users to set the folders.

A better approach is most likely using environment variables and stop looking into path for folders.

Reply:
I'm going to resolve this one since we're already discussing this in your other comments.